lib/relation-definition.js

```diff
@@ -2412,11 +2411,11 @@ EmbedsMany.prototype.prepareEmbeddedInstance = function(inst) {
 
 EmbedsMany.prototype.embeddedList =
   EmbedsMany.prototype.embeddedValue = function(modelInstance) {
-  modelInstance = modelInstance || this.modelInstance;
-  var embeddedList = modelInstance[this.definition.keyFrom] || [];
-  embeddedList.forEach(this.prepareEmbeddedInstance.bind(this));
-  return embeddedList;
-};
+    modelInstance = modelInstance || this.modelInstance;
+    var embeddedList = modelInstance[this.definition.keyFrom] || [];
+    embeddedList.forEach(this.prepareEmbeddedInstance.bind(this));
+    return embeddedList;
+  };
```

Comment:
remove indents in front from 2413 to 2418 to match how we define other funcs. ie)

```
Embeds... =
Embeds... = func() {
  ...
};
```